src/pages/User/User.js

```diff
@@ -131,6 +136,16 @@ class User extends Component {
           />
         }
         breadcrumbs={[
+          {
+            iconName: 'user',
+            text: (
+              <FormattedMessage
+                id="app.editUser.title"
+                defaultMessage="Edit user's profile"
+              />
+            ),
+            link: EDIT_USER_URI_FACTORY(userId)
```

Comment:
Edit action should not be in breadcrumbs on normal page. Instead a yellow button on left side of a page comply with our UI standards.

Reply:
If this is indeed the will of the council, Gondor will see it done!